engine/physics/src/physics/physics_3d.cpp

```diff
@@ -143,8 +143,7 @@ namespace dmPhysics
         ToBt(params.m_WorldMin, world_aabb_min, context->m_Scale);
         btVector3 world_aabb_max;
         ToBt(params.m_WorldMax, world_aabb_max, context->m_Scale);
-        int maxProxies = 1024;
```

Comment:
Why was it called maxProxies? Do you know?

Reply:
I thought about it as well, but in bullet the name of the parameter is `maxHandles`, so `proxies` some kind of replacement for `handles` which looks strange for me

Reply:
In the Box2D engine there's the concept of a proxy.